script.js

```diff
@@ -161,5 +188,6 @@ const dd = String(currentDate.getDate()).padStart(2, '0');
 const mm = String(currentDate.getMonth() + 1).padStart(2, '0');
 const yyyy = currentDate.getFullYear();
 
-today = `${yyyy}-${mm}-${dd}`;
-document.getElementById('startedOn').value = today;
+let today = `${yyyy}-${mm}-${dd}`; 
```

Comment:
is `today` being changed later? Why are we not using `const` here?

Reply:
i declared it as let just to be on the safer side.

Reply:
If we know that the variable value isn't going to change, let's keep it `const` please